docs/content/drafts/PageLayout.mdx

```diff
@@ -0,0 +1,181 @@
+---
+title: PageLayout
+status: Draft
+description: TODO
+---
+
+<Note variant="warning">Not implemented yet</Note>
+
+## Examples
+
+### Default
+
+```jsx
+<PageLayout>
+  <PageLayout.Header>Header</PageLayout.Header>
+  <PageLayout.Content>Content</PageLayout.Content>
+  <PageLayout.Pane>Pane</PageLayout.Pane>
+  <PageLayout.Footer>Footer</PageLayout.Footer>
+</PageLayout>
+```
+
+### With dividers
+
+```jsx
+<PageLayout>
+  <PageLayout.Header divider="line">Header</PageLayout.Header>
+  <PageLayout.Content>Content</PageLayout.Content>
+  <PageLayout.Pane divider="line">Pane</PageLayout.Pane>
+  <PageLayout.Footer divider="line">Footer</PageLayout.Footer>
+</PageLayout>
+```
+
+### Condensed
+
+```jsx
+<PageLayout columnGap="condensed" rowGap="condensed">
+  <PageLayout.Header>Header</PageLayout.Header>
+  <PageLayout.Content>Content</PageLayout.Content>
+  <PageLayout.Pane>Pane</PageLayout.Pane>
+  <PageLayout.Footer>Footer</PageLayout.Footer>
+</PageLayout>
+```
+
+## Props
+
+### PageLayout
+
+<!-- TODO: Resoponsive variants -->
+
+<PropsTable>
+  <PropsTableRow
+    name="containerWidth"
+    type={`| 'full'
+| 'medium'
+| 'large'
+| 'xlarge'`}
+    defaultValue="'full'"
+    description="Define the maximum width of the page container."
+  />
+  <PropsTableRow
+    name="outerSpacing"
+    type={`| 'normal'
+| 'condensed'`}
+    defaultValue="'normal'"
+  />
+  <PropsTableRow
+    name="columnGap"
+    type={`| 'normal'
+| 'condensed'`}
+    defaultValue="'normal'"
+  />
+  <PropsTableRow
+    name="rowGap"
+    type={`| 'normal'
+| 'condensed'`}
+    defaultValue="'normal'"
+  />
+  <PropsTableSxRow />
+</PropsTable>
+
+### PageLayout.Header
+
+<PropsTable>
+  <PropsTableRow name="divider" type="'none' | 'line'" defaultValue="'none'" />
+  <PropsTableRow
+    name="dividerWhenNarrow"
+    type={`| 'inherit'
+| 'none'
+| 'line'
+| 'filled'`}
+    defaultValue="'inherit'"
+  />
+  <PropsTableSxRow />
+</PropsTable>
+
+### PageLayout.Content
+
+<PropsTable>
+  <PropsTableRow
+    name="width"
+    type={`| 'full'
+| 'medium'
+| 'large'
+| 'xlarge'`}
+    defaultValue="'full'"
+    description="Define the maximum width of the content region."
+  />
+  <PropsTableSxRow />
+</PropsTable>
+
+### PageLayout.Pane
+
+<PropsTable>
+  <PropsTableRow
+    name="position"
+    type={`| 'start'
+| 'end'`}
+    defaultValue="'start'"
+  />
+  <PropsTableRow
+    name="positionWhenNarrow"
+    type={`| 'inherit'
+| 'start'
+| 'end'`}
+    defaultValue="'inherit'"
+  />
```

Comment:
Similar to divider, would be nice to merge the props?

```jsx
<PageLayout. Pane position="start"> Pane </PageLayout.Pane> 
<PageLayout. Pane position={{wide: 'end', narrow: 'start'}}> Pane </PageLayout.Pane> 
```

Reply:
Not entirely related to your comment, but some food for thought:

In the case of `PageLayout`, the only breakpoints we care about are "*when Narrow*" and "*when above Narrow*". It seems a little confusing to think of viewports as ranges but having `Medium` and `Wide` as options here -- I wouldn't expect the pane position to change from `start` on `Medium` to `end` on `Wide`, per instance.